Assets/Scripts/UI/UserInterfaceManager/UserInterfaceManager.cs

```diff
@@ -0,0 +1,46 @@
+﻿using ThatOneSamuraiGame.Scripts.UI.Pause.PauseMenu;
+using UnityEngine;
+
+namespace ThatOneSamuraiGame.Scripts.UI.UserInterfaceManager
+{
+
+    /// <summary>
+    /// Centralized gateway to user interface provider and management.
```

Comment:
Improve the comment to "Responsible for managing initialisation, state and view presentation of graphical user interfaces."